mara_moveit_config/config/mara.srdf

```diff
@@ -137,6 +141,7 @@
 
     <!--END EFFECTOR: Purpose: Represent information about an end effector.-->
     <end_effector name="moveit_ee" parent_link="ee_link" group="manipulator" />
+    <end_effector name="moveit_ee2" parent_link="ee_link" group="endeffector" />
```

Comment:
group here should be endeffector2, right?

and 

ee_link should be mara2_ee_link

Reply:
nope, mara2_ee_link is the second MARA robot. Here we only have one robot.